src/service-broker.js

```diff
@@ -964,7 +964,18 @@ class ServiceBroker {
 	 */
 	registerInternalServices(opts) {
 		opts = utils.isObject(opts) ? opts : {};
-		this.createService(require("./internals")(this), opts["$node"]);
+		const interalsSchema = require("./internals")(this);
```

Comment:
Plz fix the typo in the variable name `internalsSchema` (missing 'n')

Reply:
Fixed